pennylane_lightning/core/simulators/lightning_tensor/tncuda/bindings/LTensorTNCudaBindings.hpp

```diff
@@ -23,7 +23,6 @@
 
 #include "cuda.h"
 
-#include "BindingsBase.hpp"
```

Comment:
Any reason for moving the bindings for lightning-tensor even though we're leaving them as pybind11 currently? Is this just in prep for a change to where they will live later?

Reply:
yes